### PR TITLE
Added vote action to bill, segments and amendments

### PR DESCRIPTION
**Pull request by msfernandes**

### CODE REVIEW

**Comment by cfviotti** [comment]
Please fix any coverage tests before applying for a review. Thank you!

**Comment by msfernandes** [comment]
@cfviotti  Ok, sorry :(

**Comment by msfernandes** [comment]
@cfviotti done

**Comment by cfviotti** [comment]
Very well, you shall pass.

File: wikilegis/core/views.py [hunk 1]
```diff
@@ -66,12 +71,36 @@ def create_comment(model, segment_id, request):
...
+
+    segment.refresh_from_db()
+    html = render_to_string('segment/_votes.html', {'request': request,
+                                                    'segment': segment})
     return JsonResponse({'html': html})
 
 
 def render_new_comment(request, segment_id, segment_type):
     if request.user.is_authenticated() and request.method == 'POST':
+        print(segment_type)
```
Comment (erivanio) [inline wikilegis/core/views.py]: Please, remove the print.

**Pull request merged**